tests/map.test.cpp

```diff
@@ -1,9 +1,9 @@
 #include <libhal-util/map.hpp>
+#include <libhal-util/math.hpp>
 
 #include <limits>
 
 #include <boost/ut.hpp>
```

Comment:
```suggestion
#include <libhal-util/map.hpp>

#include <limits>

#include <libhal-util/math.hpp>

#include <boost/ut.hpp>
```

The first header should be the file under test and separate from all other headers. Grouping them will cause clang-format to reorder them alphabetically. It works out here, but can change in the future and cause uncaught include errors if the header under test doesn't come first.

Reply:
Change this for the rest of the PR.